### ISSUE TITLE
Add options to disable randomEmoji and trigger to autoopen the Picker

### ISSUE THREAD

**Issue by lKk2** [kevinfaguiar/vue-twemoji-picker#34]
Right now, there is no prop that you can set a unique Emoji or keep it static, (not have a random one everytime you hover),
it would be great if we could use even an image/icon to be the 'button',
also to be able to use it as a headless component

for example, i add my own button on click i should pass a value to the picker and only the 'popup' shows up

**Comment by kevinfaguiar** [comment]
Ok... Answering the suggested features:

- a prop called randomEmojiArray:
    - Default Value: The current uncustomizable randomEmojiArray set
    - Possible Values: emojiArray (an array of emoji unicodes: https://unicode.org/emoji/charts/full-emoji-list.html). If the array length is equal to 1 = unique static emoji; if array length more than 1, then it will randomize on hovering.

- possibility of using TwemojiPicker as an headless component: I've updated the component (version 5.2.1 onwards) for it to expose a Vue template slot (for the time being this is the maximum of a headless concept I imagine this project going, but maybe I can revisit this in the future if there is enough demand) and it is already in the docs 🥳🥳🥳: https://kevinfaguiar.github.io/vue-twemoji-picker/docs/twemoji-picker-api/#twemoji-picker-button

- as for the trigger behaviour, is can be changed with the prop triggerType: https://kevinfaguiar.github.io/vue-twemoji-picker/docs/twemoji-picker-api/#trigger-type

I will flag this issue as an enhancement while I work on the "randomEmojiArray" prop, and report back here when it is finished.

**Comment by kevinfaguiar** [comment]
The randomEmojiArray prop as detailed above is now available and documented here: https://kevinfaguiar.github.io/vue-twemoji-picker/docs/twemoji-picker-api/#randomemojiarray
Version 5.2.2 onwards

**Comment by guanzo** [comment]
`randomEmojiArray` isn't reactive 😞. I want to show 1 emoji if the picker is closed, and another when it's open. 

```html
<template>
    <TwemojiPicker ref="picker" :randomEmojiArray="triggerEmojis" />
</template>

<script>
export default {
    computed: {
        triggerEmojis () {
            return this.$refs.picker.isPickerOpen
                ? ['😄']
                : ['🙂']
        },
    },
}
</script>
```

**Comment by kevinfaguiar** [comment]
> ```
> computed: {
>         triggerEmojis () {
>             return this.$refs.picker.isPickerOpen
>                 ? ['😄']
>                 : ['🙂']
>         },
>     },
> ```

Hi there! You are right about the prop not being reactive. This was happening because I convert the Unicode to the correctly Twemoji art and wasnt updating the art on prop change. It should be fixed from version 5.5.5 onwards. 

However I would advise against the approach you are using: refs are not reactive and because of that your computed property will not be updated (it should only work IF the ref already exists). I would suggest updating your snippet to something like this (if intent is going reactive relative to a ref inner data):

```javascript
<template>
    <TwemojiPicker ref="picker" :randomEmojiArray="randomEmojiArray" />
</template>

<script>
export default {
    data() {
    return {
      content: '',
      randomEmojiArray: ['😀']
    };
  },
  mounted() {
    this.$watch(
      () => {
        return this.$refs.picker.isPickerOpen;
      },
      (value) => {
        this.randomEmojiArray = value ? ['😄'] : ['🙂'];
      }
    );
  },
}
</script>
```
